libs/surfaces/console1/console1.cc

```diff
@@ -512,6 +522,8 @@ Console1::notify_transport_state_changed ()
 void
 Console1::stripable_selection_changed ()
 {
+	if (!_in_use)
+		return;
```

Comment:
Why is this required?
It should not be possible to reach here unless the device is in use.
Perhaps just add an `assert(_in_use);`

Reply:
This was the only way of not crashing when the connnection was not set or removed. Even after my changes. It might be useless now.

Reply:
It still crashes without that check...